### PR TITLE
unlock workflow: Run the unlock animation asynchronously.

### PR DESCRIPTION
**Pull request by conte91**
Add a new animation to the unlock_bip39 workflow,
which is run asynchronously via a timer while the
seed gets unlocked.

### CODE REVIEW

File: src/workflow/unlock_bip39.c [hunk 1]
```diff
@@ -6,18 +6,74 @@
 #include <screen.h>
 #include <ui/components/trinary_input_string.h>
 #include <ui/components/ui_images.h>
+#include <ui/graphics/lock_animation.h>
 #include <ui/ugui/ugui.h>
 #include <ui/workflow_stack.h>
 #include <util.h>
 
 #ifndef TESTING
 #include <hal_delay.h>
```
Comment (benma) [inline src/workflow/unlock_bip39.c]: unused?

Comment (conte91) [reply src/workflow/unlock_bip39.c]: c599864

File: src/ui/graphics/lock_animation.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+#define LOCK_ANIMATION_FRAME_HEIGHT (25)
+#define LOCK_ANIMATION_FRAME_SIZE \
+    ((LOCK_ANIMATION_FRAME_WIDTH * LOCK_ANIMATION_FRAME_HEIGHT + 7) / 8)
+
+#include <stdint.h>
+
+/**
+ * Gets a frame of the lock animation.
+ */
+const uint8_t* get_lock_animation_frame(int frame_idx);
```
Comment (benma) [inline src/ui/graphics/lock_animation.h]: nit: y u not like namespace prefixes :sweat_smile:   `lock_animation_get_frame`

Comment (conte91) [reply src/ui/graphics/lock_animation.h]: Ehm :| Fixed.

Comment (conte91) [reply src/ui/graphics/lock_animation.h]: (c7bdccc)

**Pull request merged**